docs/docs/cmd/spe/container/container-type-add.mdx

```diff
@@ -0,0 +1,140 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo container type add
+
+Creates a new Container Type for your app
+
+## Usage
+
+```sh
+m365 spe container type add [options]
```

Comment:
let's refactor this and update the naming to `containertype`

Please check my comment from https://github.com/pnp/cli-microsoft365/issues/5767#issuecomment-2041235700 for more info